p/db-busradar-nrw/index.js

```diff
@@ -0,0 +1,119 @@
+'use strict'
+
+// DB Busradar NRW app does not allow selecting specific modes of transport to filter results,
+// so the bitmasks had to be determined by querying some stations and looking at the results..
+const products = [
+	{
+		id: 'nationalExpress',
+		mode: 'train',
+		bitmasks: [1],
+		name: 'InterCityExpress',
+		short: 'ICE',
+		default: true
+	},
+	{
+		id: 'national',
+		mode: 'train',
+		bitmasks: [2],
+		name: 'InterCity & EuroCity',
+		short: 'IC/EC',
+		default: true
+	},
+	// 4: not always used, could not be confirmed..
+	{
+		id: 'regionalExp',
+		mode: 'train',
+		bitmasks: [4],
+		name: 'Regionalexpress',
+		short: 'RE',
+		default: true
+	},
+	{
+		id: 'regional',
+		mode: 'train',
+		bitmasks: [8],
+		name: 'Regionalzug',
+		short: 'RB/RE',
+		default: true
+	},
+	// 16: often appears together with 8 even when there are only S-Bahn trains at the station
+	{
+		id: 'suburban',
+		mode: 'train',
+		bitmasks: [16],
+		name: 'S-Bahn',
+		short: 'S',
+		default: true
+	},
+	{
+		id: 'bus',
+		mode: 'bus',
+		bitmasks: [32],
+		name: 'Bus',
+		short: 'Bus',
+		default: true
+	},
+	// 64, 128, 256: could not be confirmed
```

Comment:
As above.

Reply:
64 seems to be correct, for example at Emden Außenhafen.
I can't find any station with 128 and 256 right now.